src/components/Comment/Comment.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from "react";
+import { IoMdMore } from "react-icons/io";
+import { FiEdit } from "react-icons/fi";
+import { MdDeleteOutline } from "react-icons/md";
+
+export const Comment = () => {
+  const [showOptions, setShowOptions] = useState(false);
+  return (
+    <>
+      <div className="mt-4 flex">
+        <img
+          className="h-8 rounded-full cursor-pointer"
+          src="https://res.cloudinary.com/dflebgpde/image/upload/v1652528175/socially/hacker_obpy2s.png"
```

Comment:
should provide an `alt` message for all `<img>` tags

Reply:
done